### PR TITLE
README feedback

### PR DESCRIPTION
**Pull request by malewis5**
Hi!
Here is some feedback on the README. The #1 issue is the CLI command does not create the same server application that the updated fly.toml creates leading to connection errors. I also left some comments about the changes I've made. 
Thanks!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,34 +1,31 @@
 # AnyCable Next.js/Vercel Example
```
Comment (malewis5) [inline README.md:2]: Moved the Deploy button to the top so users can quickly find it.

File: README.md [hunk 2]
```diff
@@ -1,34 +1,31 @@
...
-Learn more about AnyCable for serverless JavaScript apps in [the documentation](https://docs.anycable.io/guides/serverless).
-
-## Deployment
+This is an example of how to use [AnyCable](https://anycable.io) with [Next.js](https://nextjs.org) and [Vercel](https://vercel.com) to build real-time applications.
 
-### Prerequisites
+Learn more about AnyCable for serverless JavaScript apps in [the documentation](https://docs.anycable.io/guides/serverless).
 
-You need to deploy AnyCable-Go server to the platform of your choice. For a quick start, we recommend using [Fly.io](https://fly.io) and provide an example configuraiton.
+## Prerequisites
```
Comment (malewis5) [inline README.md:9]: Removed Deployment header in favor of Prerequisites and Deployment headers for clarity.

File: README.md [hunk 3]
```diff
@@ -1,34 +1,31 @@
...
 
-> [!NOTE]
-> AnyCable-Go can be deployed to any platform. We use Fly.io in this example because it's easy to deploy and manage.
+You will need to deploy an AnyCable-Go server to the platform of your choice. We recommend using [Fly.io](https://fly.io) and provide an example configuration in `fly.toml.example` for seamless deployments. However, Anycable-Go server can be deployed on any platform.
```
Comment (malewis5) [inline README.md:11]: Removed all > [!NOTE] references. The templates page pulls in your README and these do not render correctly.

File: README.md [hunk 4]
```diff
@@ -1,34 +1,31 @@
...
 
 Using [fly CLI](https://fly.io/docs/hands-on/install-flyctl/), run the following command to create and launch a new AnyCable-Go application:
 
 ```sh
 # Create a new Fly application
 fly launch --image anycable/anycable-go:1.4 --generate-name --ha=false --internal-port 8080 --env PORT=8080 --env ANYCABLE_PRESETS=fly,broker
 ```
+> 🚨 This CLI command is incorrect 🚨
```
Comment (malewis5) [inline README.md]: This needs to be updated and then we will remove this line.

File: README.md [hunk 5]
```diff
@@ -1,34 +1,31 @@
...
+
 
-Answer all the questions ("No" to all database-related questions, "Yes" to deployment). In the end, you will a `fly.toml` file with the minimal configuration for your app. See also [fly.toml.example](./fly.toml.example) to learn more about other available (and recommended) configuration options.
+Answer "No" to all database-related questions and "Yes" to deployment. This will deploy your app and create a `fly.toml` file with the minimum configuration. See the `fly.toml.example` file to learn more about other available and recommended configuration options.
```
Comment (malewis5) [inline README.md:20]: Edited for clarity.

File: README.md [hunk 6]
```diff
@@ -37,9 +34,10 @@ Answer all the questions ("No" to all database-related questions, "Yes" to deplo
   ANYCABLE_JWT_ID_KEY=<YOUR_JWT_SECRET>
   ```
 
-  Feel free to use any strings as secrets (but don't forget to keep them in a safe and secure place).
+  * The `ANYCABLE_HTTP_BROADCAST_SECRET` and `ANYCABLE_JWT_ID_KEY` can be any strings.
```
Comment (malewis5) [inline README.md:35]: Added this so users can easily create a secure value.

File: README.md [hunk 7]
```diff
@@ -53,23 +51,25 @@ When using Fly, you can keep all env vars in the `.env.production` file and impo
 cat .env.production | fly secrets import
 ```
 
-In the Vercel app configuration, you can copy-paste the contents of the `.env.production` file (or import it) on the Environment Variables configuration page.
+When deploying to Vercel you can use the [Vercel CLI](https://vercel.com/docs/cli) to pull environment variables:
```
Comment (malewis5) [inline README.md:52]: Added this so users can pull .env variables using our CLI.

File: README.md (hunk 4 above)
Comment (palkan) [inline README.md]:
```suggestion
fly launch --image anycable/anycable-go:1.4 --generate-name --ha=false --internal-port 8080 --env PORT=8080 --env ANYCABLE_BROKER=memory
```
```

Fixed!

**Review by palkan — APPROVED** [review]
Thanks, Matt!

Updated the launch command (will fix the markup after merge); everything else looks good

**Pull request merged**